src/target/common/stm32/clock.c

```diff
@@ -219,46 +219,64 @@ void CLOCK_ClearMsecCallback(int cb)
     msec_callbacks &= ~(1 << cb);
 }
 
+// Run Mixer one time.  Used by protocols that trigger mixer calc in protocol code
+static volatile int *mixer_sync;
+void CLOCK_RunMixer(volatile int *sync) {
```

Comment:
Just avoiding making mixer_sync a global since it must be accessed from within the protocols.  But we have plenty of other globals so that might fit better.  I'll consider this change.

Reply:
Using a global for mixer_sync does make the code look cleaner I think.  Pushed the change. Good suggestion.